playwright-test/multi-page/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "multi-page",
+  "private": true,
+  "scripts": {
+    "test": "playwright test"
+  },
+  "devDependencies": {
+    "@axe-core/watcher": "^2.0.0-next.099d5645",
```

Comment:
Support for `@playwright/test` won't be in a stable version until v2.1.0. We **must** use the `@next` channel.